[2024-07-02T02:02:45.889+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-02T02:02:45.906+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_minio_s3_v03.sensor_minio_s3 scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T02:02:45.915+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_minio_s3_v03.sensor_minio_s3 scheduled__2024-07-01T00:00:00+00:00 [queued]>
[2024-07-02T02:02:45.915+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-07-02T02:02:45.933+0000] {taskinstance.py:2330} INFO - Executing <Task(S3KeySensor): sensor_minio_s3> on 2024-07-01 00:00:00+00:00
[2024-07-02T02:02:45.941+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_minio_s3_v03', 'sensor_minio_s3', 'scheduled__2024-07-01T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_minio_s3.py', '--cfg-path', '/tmp/tmpf8ou6pg6']
[2024-07-02T02:02:45.943+0000] {standard_task_runner.py:91} INFO - Job 8: Subtask sensor_minio_s3
[2024-07-02T02:02:45.944+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=5266) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-02T02:02:45.945+0000] {standard_task_runner.py:63} INFO - Started process 5268 to run task
[2024-07-02T02:02:45.992+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_minio_s3_v03.sensor_minio_s3 scheduled__2024-07-01T00:00:00+00:00 [running]> on host 468cd84e5e4e
[2024-07-02T02:02:46.079+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='vohoang' AIRFLOW_CTX_DAG_ID='dag_with_minio_s3_v03' AIRFLOW_CTX_TASK_ID='sensor_minio_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-07-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-01T00:00:00+00:00'
[2024-07-02T02:02:46.080+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-02T02:02:46.094+0000] {baseoperator.py:400} WARNING - S3KeySensor.execute cannot be called outside TaskInstance!
[2024-07-02T02:02:46.095+0000] {s3.py:117} INFO - Poking for key : s3://***/data.csv
[2024-07-02T02:02:46.104+0000] {base.py:84} INFO - Using connection ID 'minio_s3_conn' for task execution.
[2024-07-02T02:02:46.104+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='minio_s3_conn', conn_type='aws') credentials retrieved from login and password.
[2024-07-02T02:02:46.417+0000] {base.py:294} INFO - Success criteria met. Exiting.
[2024-07-02T02:02:46.418+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-02T02:02:46.426+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_with_minio_s3_v03, task_id=sensor_minio_s3, run_id=scheduled__2024-07-01T00:00:00+00:00, execution_date=20240701T000000, start_date=20240702T020245, end_date=20240702T020246
[2024-07-02T02:02:46.479+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-02T02:02:46.494+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-02T02:02:46.499+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
